pdpdmeetup/templates/home.html

```diff
@@ -0,0 +1,114 @@
+{% extends "base.html" %}
+
+{#{% block tabtitle %} {{ tittle }} {% endblock tabtitle %}#}
+
+{% load staticfiles %}
+
+{% block javascript %}
+    <script>
+        $(document).ready(function()
+        {
+            // Load any meetups with a status of 'upcoming' on page load
+            $.ajax({
+                type: "GET",
+                url: "{% url 'homeapp:ajax_meetups_tab' 'upcoming' %}",
+                error: function(data){
+                    $('#upcoming-meetups-tab-content').html("Ooops. I couldn't get the meetup info.");
+                    },
+                success: function(data){
+                    $('#upcoming-meetups-tab-content').html(data);
+                }
+            })
+        })
+    </script>
+
+    <script>
+        $(document).ready(function()
+        {
+            // Load any meetups with a status of 'upcoming' on page load
+            $.ajax({
+                type: "GET",
+                url: "{% url 'homeapp:ajax_meetups_tab' 'past' %}",
+                error: function(data){
+                    $('#past-meetups-tab-content').html("Ooops. I couldn't get the meetup info.");
+                    },
+                success: function(data){
+                    $('#past-meetups-tab-content').html(data);
+                }
+            })
+        })
+    </script>
+
+{% endblock %}
+
+{% block content %}
+
+	<h1>{{ group_name }}</h1>
+
+    <div class="row">
+        <div class="col-md-12">
+            <div class="jumbotron">
+                <div class="row">
+                    <div class="col-md-8">
+                        <div class="text-center center-block">
+                            <h2>Hey Python & Django fans!</h2>
+                            <p>A meetup group for Perthfectionists with deadlines</p>
+                        </div>
+                    </div>
+
+                    <div class="col-md-4">
+                        <div class="row">
+                            <div class="text-center center-block">
+                                <h6 class="text-muted">Our Awesome Sponsors</h6>
```

Comment:
List of sponsors should be dynamically generated from the Company model. Company might need a new field called `is_sponsor`.